service/src/ConstConfigIOGroup.hpp

```diff
@@ -88,6 +88,7 @@ namespace geopm
                 int domain;
                 std::function<double(const std::vector<double> &)> agg_function;
                 std::string description;
+                bool default_value;
```

Comment:
Nit on naming.  Default value is a boolean indicating if a devalue value exists but might be interpreted as the actual default value.  Default_Value_provided is used elsewhere as and seems somewhat clearer...or maybe default_value_exists or default_value_present?  

Otherwise lgtm.